samples/graph-activity-feed/csharp/ActivityFeedNotification/Scripts/teamsapp.js

```diff
@@ -2,7 +2,7 @@
   'use strict';
 
   // Call the initialize API first
-  microsoftTeams.initialize();
+  microsoftTeams.app.initialize();
 
   // Check the initial theme user chose and respect it
   microsoftTeams.getContext(function (context) {
```

Comment:
`getContext` and the microsoftTeams settings below are still using v1 functions

Reply:
Removed the file as it was not used